### DISCUSSION TITLE
Prettier для проекта

### DISCUSSION THREAD

**Discussion by edbond88** [doka-guide/platform#797] [Идеи] [answered]
Видел в [соседнем треде](https://github.com/doka-guide/platform/discussions/205#discussioncomment-887563) предложение внедрить prettier. 

Могу заняться этим вопросом, но нужно утвердить тогда конфиг prettier.

Призываю мейтейнеров проекта

**Comment by pepelsbey** [comment]
Спасибо! Обсудим на дев-созвоне и вернёмся :)

**Comment by nlopin** [comment, marked as answer]
Привет! мы обсудили и создали задачу #825. Приходи делать, будем очень рады твоей помощи

**Reply by edbond88** [reply]
Отлично, беру!
